woocommerce/woocommerce
# Fix `get_options` deprecation notice

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



This PR fixes the same issue in https://github.com/woocommerce/woocommerce/issues/32863.

We deprecated the `get_options` function in https://github.com/woocommerce/woocommerce-admin/pull/8279/ and only allowed API requests to fetch options in woocommerce_permissions list. We still forgot to update the allowed list when we used them in the frontend. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Use a JN site
2. Go through WC pages
3. Confirm no `get_options` deprecation notice in the PHP error log
